api/core/model_runtime/model_providers/bedrock/bedrock.py

```diff
@@ -18,9 +19,13 @@ def validate_provider_credentials(self, credentials: dict) -> None:
         try:
             model_instance = self.get_model_instance(ModelType.LLM)
 
-            # Use `gemini-pro` model for validate,
+            default_bedrock_validate_model_name = 'amazon.titan-text-lite-v1'
+            bedrock_validate_model_name = os.getenv('AWS_BEDROCK_VALIDATE_MODEL')
```

Comment:
Thanks for the PR. Could you add an input field to the credential form instead and have something like
`bedrock_validate_model_name = credentials.get('model_for_validation', 'amazon.titan-text-lite-v1')`?

Here's how it might look:
```
  - variable: model_for_validation
      required: false
      label:
        en_US: Available Model Name
        zh_Hans: 可用模型名称
      type: text-input
      placeholder:
        en_US: A model you have access to (e.g. amazon.titan-text-lite-v1) for validation.
        zh_Hans: 为了进行验证，请输入一个您可用的模型名称 (例如：amazon.titan-text-lite-v1)
```

Reply:
okay, thank you for your suggestion @guchenhe